SHAPETEST/StartShapeTestBox.md

```diff
@@ -6,9 +6,14 @@ aliases: ["0xFE466162C4401D18"]
 
 ```c
 // 0xFE466162C4401D18 0x249BC876
-int START_SHAPE_TEST_BOX(float x, float y, float z, float x1, float y2, float z2, float rotX, float rotY, float rotZ, Any p9, Any p10, Any entity, Any p12);
+int START_SHAPE_TEST_BOX(float x, float y, float z, float x1, float y2, float z2, float rotX, float rotY, float rotZ, Any p9, int flags, Entity entity, Any p12);
 ```
 
+```
+You can use _GET_RAYCAST_RESULT to get the result of the raycast.
```

Comment:
Could you link to the function? Markdown style `[text](link)` works. Same for the other function mention.